templates/actor/parts/actor-combat.html

```diff
@@ -96,7 +96,7 @@
         </div>
 
         <div class="flexrow flex-group-left">
-             <label class="item-title flex-group-left rollable" data-roll="damage" data-name='{{localize "arm5e.sheet.damage"}}' data-option1="{{ data.characteristics.str.value }}" data-txtoption1='{{ localize "arm5e.sheet.strength" }}' data-option2="{{ data.combat.dam }}" data-txtoption2='{{localize "arm5e.sheet.damage"}}'>{{localize "arm5e.sheet.damage"}}:</label>
+            <label class="item-title flex-group-left rollable" data-roll="damage" data-name='{{localize "arm5e.sheet.damage"}}' data-option1="{{ data.characteristics.str.value }}" data-txtoption1='{{ localize "arm5e.sheet.strength" }}' data-option2="{{ data.combat.dam }}" data-txtoption2='{{localize "arm5e.sheet.damage"}}'>{{localize "arm5e.sheet.damage"}}:</label>
             <!-- str + weapon + attack ventage -->
```

Comment:
I double checked, there is no roll for damage. You add your strength and weapon damage modifier and that's all, the randomness part already occured when rolling for attack/defense

Reply:
Crap, let me check that.

Reply:
FIxed I would say.